configuration/frontend/base-config.json

```diff
@@ -31,5 +31,17 @@
       "generalPatientNoteUuid": "3cd9d956-26fe-102b-80cb-0017a47871b2",
       "midUpperArmCircumferenceUuid": "4326b04b-3158-417a-bb8d-ad022295b0f4"
     }
+  },
+  "@openmrs/esm-patient-medications-app": {
+    "durationUnitsConcept": "3cd706b8-26fe-102b-80cb-0017a47871b2",
+    "daysDurationUnit": {
+      "uuid": "3cd706b8-26fe-102b-80cb-0017a47871b2"
```

Comment:
This is another example of a configuration property that shouldn't need to be set/maintained here, as elsewhere in OpenMRS the various duration units are mapped to standard SNOMED-CT duration codes that can be interpreted (see BaseDosingInstructions.java and Duration.java in OpenMRS core).  This may be outside the scope of this PR, and this may well be the correct configuration, but we should challenge whether this should need to be configured here.